lib/gooddata/lcm/actions/update_release_table.rb

```diff
@@ -39,6 +39,7 @@ class UpdateReleaseTable < BaseAction
       class << self
         def call(params)
           client = params.gdc_gd_client
+          GoodData.gd_logger.info('Updating release table in NFS') unless params.ads_client
```

Comment:
Use splunk style action=update_release_table use_nfs=true/false

Reply:
@phong-nguyen-duy Please take a look and update code for this log. Thank you.

Reply:
Done fix it.

Reply:
updated